.gitignore

```diff
@@ -0,0 +1,5 @@
+npm-debug.log
+node_modules
+coverage
+build
```

Comment:
❓ Is there an .npmignore? If not this will be used and build will not be included

Reply:
Forgot to add it. Done